server/channels/app/channel.go

```diff
@@ -1668,8 +1668,11 @@ func (a *App) AddChannelMember(c request.CTX, userID string, channel *model.Chan
 			return nil, err
 		}
 	} else {
+		crctx := c.Clone()
```

Comment:
Nit: I must say I'm not a fan of this naming, it took me a while to understand it meant cloned request context.